src/app/participant-page/participant-page.component.html

```diff
@@ -514,14 +514,14 @@ <h3 class="Color--warn Line--Break">Lost to Follow-Up</h3>
                       <tr>
                         <td class="TD--Padding Width--50 Data-Display-Question">Sent:</td>
                       </tr>
-                      <tr>
+                      <tr *ngIf="sample.scanDate !== 0">
```

Comment:
can you add a message or '-' for scanDate === 0
thank you

(same for receiveDate === 0)

Reply:
sure, added